wal/system.py

```diff
@@ -15,16 +15,23 @@ def idle_sec(self) -> int:
 class LinuxX(SystemInterface):
     envs = {'DISPLAY': ":0"}
 
-    def active_window(self) -> str:
+    def active_window(self) -> tuple[str, str]:
         active_window_id = cmd_output(
             "xprop -root 32x '\t$0' _NET_ACTIVE_WINDOW | cut -f 2", envs=self.envs
         ).strip()
         active_window = cmd_output(
             f"xprop -id {active_window_id} _NET_WM_NAME", envs=self.envs
         )
-        active_window = active_window.replace('_NET_WM_NAME(UTF8_STRING) = "', "")
-        active_window = active_window.strip()[:-1]
-        return active_window
+        active_window = active_window.replace("_NET_WM_NAME(UTF8_STRING) = ", "")
+        active_window = active_window.strip('\t\n "')
```

Comment:
Fancy usage of `strip` ^^.. `strip` can be passed a string with characters to be stripped :smile: